cves/CVE-2013-0306.yml

```diff
@@ -77,13 +81,24 @@ vccs:
 - commit: 7adffaeaf6dfa22db0b6b2a29632b9150c7ac732
   note: This VCC was discovered automatically via archeogit.
 - commit: aba95dcc0b5370ffac3d3b701c3ca7782ee999c1
-  note: This VCC was discovered automatically via archeogit.
+  note: |
+    This VCC was discovered automatically via archeogit.
+    Removed ambiguity with regard to the max_num option of
+    formsets and as a result of admin inlines.
 - commit: 65b380e74ab79ee011f7556e342dd4d656b018a5
-  note: This VCC was discovered automatically via archeogit.
+  note: |
+    This VCC was discovered automatically via archeogit.
+    In Django 1.3 ``FormSet`` creation behavior is modified slightly. Historically
+    the class didn't make a distinction between not being passed data and being
+    passed empty dictionary. This was inconsistent with behavior in other parts of
+    the framework. Starting with 1.3 if you pass in empty dictionary
+    the ``FormSet`` will raise a ``ValidationError``.
 - commit: 8bafde1229fdebb48383449de9bcadde06451816
   note: This VCC was discovered automatically via archeogit.
 - commit: e061b036a54597b6a994573ea6aa2ee02ca8838e
-  note: This VCC was discovered automatically via archeogit.
+  note: |
+    This VCC was discovered automatically via archeogit.
+    Ensure formsets evaluate to True even if they have no forms.
 - commit: 386b12c1c668dd54d24704ef974e17b38abb825d
   note: This VCC was discovered automatically via archeogit.
```

Comment:
Please remove any VCCs that did not directly contribute to the issue (though keeping any that indirectly contributed to the issue in a useful-to-study way would be desirable as well). If they all contributed, please note the way that they contributed, as many of the VCCs are pretty indistinguishable right now.